activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Prevent merging target_scope conditions for *_through with STI case
```

Comment:
People who read the change log do will not be reading the test code. It is intended for users or casual contributers, not the core team. You need to list why a change was made in the changelog for something to be merged.